x-pack/plugins/reporting/server/core.ts

```diff
@@ -26,12 +28,13 @@ import { screenshotsObservableFactory, ScreenshotsObservableFn } from './lib/scr
 import { ReportingStore } from './lib/store';
 
 export interface ReportingInternalSetup {
+  basePath: Pick<BasePath, 'set'>;
```

Comment:
`basePath.get` was actually never being called in Reporting. Instead, we've been getting the base path (for full URLs) using `config.kbnConfig.get('server', 'basePath')`